mision_to_mars/src/mision_to_mars/Simulation.java

```diff
@@ -1,146 +1,130 @@
-import java.util.Iterator;
-import java.util.List;
 import java.util.Scanner;
 
 /**
- * Created by ralphemerson on 11/23/2017.
+ * This class consists in the simulation for two rockets.
+ * This class contains methods creates a simulation
+ *
+ * @author Andres Uzeda
+ * @since February 6, 2019
+ * @version 1.0
  */
 public class Simulation {
-    private ArrayList<Item> items;
-    private ArrayList<Rocket> rocketU1;
-    private ArrayList<Rocket> rocketU2;
-    private boolean hasLanded = true;
-    private File phase1 = new File("phase-1.txt");
-    private File phase2 = new File("phase-2.txt");
 
-    public Simulation() {
-    }
-
-    public ArrayList<Item> loadItems(int phase) throws FileNotFoundException {
-
-        if (phase == 1) {
-            System.out.println("Loading phase 1");
-            loadPerPhaseItems(phase1);
-            System.out.println("Done loading phase 1");
-        } else {
-            System.out.println("Loading phase 2");
-            loadPerPhaseItems(phase2);
-            System.out.println("Done loading phase 2\n");
-        }
-
-        return items;
-    }
-
-    public void loadPerPhaseItems(File file) throws FileNotFoundException {
-        items = new ArrayList<>();
-        Scanner fileScanner = new Scanner(file);
-
-        while (fileScanner.hasNextLine()) {
-            String string = fileScanner.nextLine();
-            String[] separated = string.split("=");
-
-            Item item = new Item(separated[0], Integer.parseInt(separated[1]));
-            items.add(item);
+    /**
+     * This method returns a list of items in a document
+     * path e.g "items_to_load/phase-1.txt"
+     * @return a list of items
+     */
+    public ArrayList<Item> readData(String path){
+        try {
+            return getAlistItem(path);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
         }
+        return null;
     }
 
-    public ArrayList<Rocket> loadU1(List<Item> itemsU1) {
-        System.out.println("loading U1...");
-        rocketU1 = new ArrayList<>();
-        Rocket rocket = new U1();
-        Iterator iterator = itemsU1.iterator();
-
-        while (iterator.hasNext()) {
-            Item item = (Item) iterator.next();
-            if (rocket.canCarry(item)) {
-                rocket.carry(item);
-            } else {
-                rocketU1.add(rocket);
-                rocket = new U1();
-                System.out.println("New U1 rocket created");
-                rocket.carry(item);
-            }
-            if (!iterator.hasNext()) {
-                rocketU1.add(rocket);
+    /**
+     * This method returns a list of items in a rocket
+     * @param rocket
+     * @param listOfItemsToCarry
+     * @return this method returns a list of items loaded in a rocket
+     */
+    public ArrayList<Item> loadItems(Rocket rocket, ArrayList<Item> listOfItemsToCarry){
+        System.out.println("Adding Items to "+ rocket.getClass().getTypeName());
```

Comment:
please use other method of concatenation for example:

"Adding Items to ".concat(rocket.getClass().getTypeName());